shared/common/hooks/useKeyboardShortcut.ts

```diff
@@ -0,0 +1,24 @@
+import {useEffect} from "react";
+
+export function useKeyboardShortcut(key: string, callback: Function) {
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      const isMac = navigator.platform.toUpperCase().indexOf("MAC") >= 0;
+      const isKeyPressed = event.key.toLowerCase() === key.toLowerCase();
```

Comment:
```suggestion
      const isKeyPressed = event.key === key;
```
I don't think `toLowerCase` is needed here. `Cmd+P` and `Cmd+Shift+P` are different shortcuts, but lowercasing would treat them the same.